docs/src/main/asciidoc/README.adoc

```diff
@@ -44,5 +44,6 @@ complete list of externalized variables for use is given in the following table:
 |\{quickstarts-blob-url}|{quickstarts-blob-url}| Quickstarts URL to master blob source tree; used for referencing source files.
 |\{quickstarts-tree-url}|{quickstarts-tree-url}| Quickstarts URL to master source tree root; used for referencing directories.
 
-|\{graalvm-version}|{graalvm-version}| Recommended Graal VM version to use.
+|\{graalvm-version}|{graalvm-version}| Recommended GraalVM version to use.
+|\{graalvm-flavor}|{graalvm-flavor}| The full flavor of GraaVM to use e.g. `19.3.1-java11`.
```

Comment:
This is unrelated to this PR, but why do we need two columns (`Property Name` and `Value`) while they contain the exact same content for all lines of the table?

Reply:
They do not. One is escaped, not the other.

Reply:
Oh I missed that detail, sorry :)

Reply:
I can see the HTML with different values in `quarkus-documentation-999-SNAPSHOT.zip`, but is this file actually published somewhere on a public website? I can't find it anywhere on `https://quarkus.io/`.

Reply:
On GH, there isn't any difference in terms of display betweens columns 1 and 2.

Reply:
No, it's not. It's just for reference, we don't publish it.

That would be a GH bug :).

Reply:
Then I still don't understand why we need the `Value` column.